src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/ToHtml.kt

```diff
@@ -0,0 +1,31 @@
+package nl.avisi.structurizr.site.generatr.site.model
+
+import com.structurizr.documentation.Format
+import kotlinx.html.div
+import kotlinx.html.stream.createHTML
+import nl.avisi.structurizr.site.generatr.site.views.diagram
+import org.jsoup.nodes.Element
+
+const val embedPrefix = "embed:"
+fun toHtml(format: Format, pageViewModel: PageViewModel, content: String, svgFactory: (key: String, url: String) -> String?): String = when(format) {
+    Format.AsciiDoc -> asciidocToHtml(pageViewModel, content, svgFactory)
+    Format.Markdown -> markdownToHtml(pageViewModel, content, svgFactory)
+    else            -> markdownToHtml(pageViewModel, content, svgFactory) // ensures past behaviour
```

Comment:
Can we reach the else branch here in real life? Generating "unsupported document" like above would be an alternative.